packages/oidc-http-server-pages/src/get-static-page.ts

```diff
@@ -0,0 +1,61 @@
+import type { ITemplate, HttpServerPage, PageTemplates } from './types';
+
+function findTemplate(templates: ITemplate[], parameterKeys: string[]) {
+  const parametersJoined = parameterKeys.sort().join('-');
+  for (const template of templates) {
+    const templateParametersJoined = Object.keys(template.parameters)
+      .sort()
+      .join('-');
+    if (parametersJoined === templateParametersJoined) return template;
+  }
+}
+
+function replacePlaceholders(
+  template: ITemplate,
+  parameters: Record<string, string | undefined>
+) {
+  let { html } = template;
+  for (const [key, placeholder] of Object.entries(template.parameters)) {
+    html = html.replaceAll(placeholder, escapeHTML(parameters[key] as string));
+  }
+  return html;
+}
+
+function escapeHTML(str: string) {
```

Comment:
is it a best practice to use explicit return types always? I thought with simple ones like this there is no need. a quick search doesn't tell me otherwise

Reply:
Yeah, I'd say so – I thought we had https://typescript-eslint.io/rules/explicit-function-return-type/ enabled in a few places, but I guess it didn't make it into the "shared" eslint config in this repo. I'd generally recommend always doing this, yes, unless the type is hard to spell out and inconsequential, otherwise it's a good idea to check that you return the type that you're actually expecting to return